index.bs

```diff
@@ -6797,9 +6797,7 @@ This [=client extension|client=] [=registration extension=] facilitates reportin
 ::  None, other than to report on credential properties in the output.
 
 : Client extension output
-:: [=map/Set=] <code>[=credentialCreationData/clientExtensionResults=]["{{AuthenticationExtensionsClientOutputs/credProps}}"]["rk"]</code> to the value of the |requireResidentKey| parameter that was used in the <a href='#CreateCred-InvokeAuthnrMakeCred'>invocation</a> of the [=authenticatorMakeCredential=] operation.
```

Comment:
Perhaps it's worth keeping this as a recommendation for what to do during a registration ceremony? It should probably move into the **rk** section below in that case.

Reply:
After considering your feedback and re-reading the line I removed, I kinda want to move this into the "Client extension processing" section above as it reads like instructions for the client.

For developers I suggest that the existing explanation of `credProps.rk` is sufficient...

> If [rk](http://localhost:8000/#dom-credentialpropertiesoutput-rk) is true, the credential is a [discoverable credential](http://localhost:8000/#discoverable-credential). If [rk](http://localhost:8000/#dom-credentialpropertiesoutput-rk) is false, the credential is a [server-side credential](http://localhost:8000/#server-side-credential). If [rk](http://localhost:8000/#dom-credentialpropertiesoutput-rk) is not present, it is not known whether the credential is a [discoverable credential](http://localhost:8000/#discoverable-credential) or a [server-side credential](http://localhost:8000/#server-side-credential).

...And removing this line allows for the overall page structure of "Client extension output" for `credProps` to match the structure of this same section in the entries for `prf` and `largeBlob`.

What are your thoughts on this? 🤔

Reply:
Sounds fair, I think? Please go ahead, and let's see how that looks as a whole.

Reply:
I've gone ahead and restored this line to the "Client extension processing" section for our consideration.